### ISSUE TITLE
The CSS in the Risk Input Preparation Toolkit is broken

### ISSUE THREAD

**Issue by daniviga** [gem/oq-platform#472]
- OQ logo misplaced
- menu items bottom offset wrong 
- username sub-menu arrow is too low
- the main content is slightly shifted to the left (see the tile in the screenshots)
- the footer is broken (too narrow)

![bad](https://cloud.githubusercontent.com/assets/1818657/10069205/e2ce8f84-62aa-11e5-8ae7-4f8a36cb7b0b.png)

should be:

![good](https://cloud.githubusercontent.com/assets/1818657/10069209/e741c892-62aa-11e5-9733-bf887125a7ef.png)

The result is the same with FF and Chrome

**Comment by daniviga** [comment]
The issue seems caused by the import of a newer version of bootstrap which conflicts with the bootstrap used by GeoNode (part of integrated in assets.min.css):

```html
<link href="/static/css/bootstrap-v3.3.5.css" rel="stylesheet">
```

you can see that also the render of buttons is strange (left as it is now):
![button](https://cloud.githubusercontent.com/assets/1818657/10069567/49270d18-62ad-11e5-8778-d7390f718994.png)

**Comment by daniviga** [comment]
Fixed in #478
